spec/ameba/rule/lint/empty_ensure_spec.cr

```diff
@@ -61,7 +61,9 @@ module Ameba::Rule::Lint
       issue = s.issues.first
 
       issue.rule.should_not be_nil
-      issue.location.to_s.should eq "source.cr:2:3"
+      # TODO: refactor this in next release
+      # Crystal 1.4 changes the start location of Crystal::ExceptionHandler
+      # issue.location.to_s.should eq "source.cr:2:3"
```

Comment:
This could be wrapped with conditional check.

Reply:
I don't see too much profit since I would like to re-work the way the location is returned for these two cases. However, if you think it is required, I can add it.

Reply:
If we're to go with BC this would be a good thing IMO. OTOH I'd personally drop it and make ameba v0.15 require Crystal v1.4.

Reply:
The last release was prepared a while ago and we now have a big set of changes in the pipeline. I think BC would be useful for people who face regressions on v0.15 

Added a check

Reply:
I don't follow. BC doesn't change anything for people facing regressions, since there's no way they can go back to pre-0.15 since it doesn't work with the newest Crystal.

Reply:
If no BC:

1. Shard moves to Crystal 1.4
2. Shard is forced to move to Ameba 0.15
3. Let's say there is a regression on 0.15, as a result crystal upgrade is blocked 

In my understanding, BC avoids this. What do I miss?

Reply:
Yes, you are right